src/flanders/core.cljc

```diff
@@ -0,0 +1,176 @@
+(ns flanders.core
+  (:refer-clojure :exclude [int key keyword map name num str type])
+  (:require #?(:clj  [clojure.core :as core]
+               :cljs [cljs.core :as core])
+            [clojure.zip :as z]
+            [flanders.types :as ft]
+            [schema.core :as s]))
+
+;; ----------------------------------------------------------------------
+;; Defining Branch Nodes
+;; ----------------------------------------------------------------------
+
+(declare key)
+(declare enum)
+(declare seq-of)
+
+(defn entry [key_ type_ & {:keys [description reference required?]
+                         :or {required? true}}]
+  (let [key_ (if (keyword? key_) (key key_) key_)
```

Comment:
Just a note to remark it took me long time to understand it wasn't `clojure.core/key` even if the declare is just above.